.travis.yml

```diff
@@ -43,13 +43,13 @@ jobs:
       env:
       - PURPOSE='Automation'
       - REDUCE_SDK='True'
-      - AZURE_CLI_TEST_TARGET_PROFILE='2017-03-09'
+      - AZURE_CLI_TEST_TARGET_PROFILE='2017-03-09-profile'
```

Comment:
Why? We want to run `latest` profile as well. This should be an additional job instead of changing the current job.